src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoTicket {
+
+    private final List<LottoNumbers> tickets = new ArrayList<>();
+
+    public LottoTicket(final int ticketSize,
+                       final LottoNumberSelectPolicy lottoNumberSelectPolicy) {
+        for (int i = 0; i < ticketSize; i++) {
+            tickets.add(new LottoNumbers(lottoNumberSelectPolicy.selectNumbers()));
+        }
+    }
+
+    public int size() {
+        return tickets.size();
+    }
+
+    public List<LottoNumbers> getTickets() {
+        return Collections.unmodifiableList(tickets);
+    }
+
+    public List<LottoRank> getLottoRanks(final WinningNumbers winningNumbers) {
+        List<LottoRank> lottoRanks = new ArrayList<>();
+        for (LottoNumbers lottoNumbers : tickets) {
+            lottoRanks.add(winningNumbers.match(lottoNumbers));
+        }
+        return lottoRanks;
+    }
```

Comment:
```suggestion
    public LottoResult getLottoRanks(final WinningNumbers winningNumbers) {
        List<LottoRank> lottoRanks = new ArrayList<>();
        for (LottoNumbers lottoNumbers : tickets) {
            lottoRanks.add(winningNumbers.match(lottoNumbers));
        }
        return new LottoResult(lottoRanks);
    }
```
https://github.com/next-step/java-lotto/pull/2029/files#r701206704 요런 의미였습니다